integrations/slack/src/handler.ts

```diff
@@ -33,7 +34,9 @@ export const handler: botpress.IntegrationProps['handler'] = async ({ req, ctx,
     const actionValue = await respondInteractive(body)
 
     if (body.type !== 'block_actions') {
-      throw Error(`Interaction type ${body.type} is not supported yet`)
+      const errMessage = `Interaction type ${body.type} is not supported yet`
```

Comment:
suggestion, just for extra clarity

```suggestion
      const errMessage = `Interaction type ${body.type} received from Slack is not supported yet`
```